README.md

```diff
@@ -47,6 +47,20 @@ The Flexy's clock must be set to Coordinated Universal Time (UTC) in order to po
 
 Using FTP transfer the contents of the osisoft_connector directory to the /usr/ directory of your eWON via FTP.  If successful you should have a css directory, js directory, config.html, ConnectorConfig.json, flexy-osisoft-connector.jar, and jvmrun located in the /usr/ directory of the Flexy.  You must also follow all steps in the Certificates section below.
 
+### Tag's Configuration
+
+The tag's historical logs are the source for the data pushed to OSIsoft, so settings for your historical logging will effect what data can get sent to OSIsoft. Make sure you enable historical logging in your tag settings. 
++ On the flexy web page, navigate to Tags -> values
++ On the web top left section of the page, switch the mode from view to setup
++ Click on your tag that you wish to configure, and click on edit
++ On the tag configuration panel scroll down and enable Historical Logging
+
+We suggest you make use of both the available historical logging settings: Logging Deadband and Logging Interval. More information on Logging DeadBand is listed below.
```

Comment:
we - first person

Consider rephrasing to something like
It is suggested to make use ...